src/ontodia/diagram/model.ts

```diff
@@ -403,6 +412,9 @@ export class DiagramModel {
             id: linkTypeId,
             label: [{text: uri2name(linkTypeId), lang: ''}],
         });
+
+        this.initLinkSettings([linkType], this.linkSettings);
```

Comment:
`initLinkSettings` should be called only once on importing layout, not on every `createElement` call.